packages/workbox-core/test/bundle/node/utils/test-logHelper.js

```diff
@@ -0,0 +1,339 @@
+import chai from 'chai';
+import sinon from 'sinon';
+
+import logHelper from '../../../../src/utils/logHelper';
+
+const expect = chai.expect;
+
+describe(`logHelper [${process.env.NODE_ENV}]`, function() {
+  let stubs = [];
+
+  beforeEach(function() {
+    logHelper.logLevel = logHelper.LOG_LEVELS.verbose;
+  });
+
+  afterEach(function() {
+    stubs.forEach((stub) => {
+      stub.restore();
+    });
+
+    stubs = [];
+  });
+
+  describe(`LOG_LEVELS`, function() {
+    it(`should expose the valid LOG_LEVELS`, function() {
+      expect(logHelper.LOG_LEVELS).to.exist;
+    });
+
+    it(`should expose the expected LOG_LEVELS`, function() {
+      expect(logHelper.LOG_LEVELS.verbose).to.exist;
+      expect(logHelper.LOG_LEVELS.debug).to.exist;
+      expect(logHelper.LOG_LEVELS.warning).to.exist;
+      expect(logHelper.LOG_LEVELS.error).to.exist;
+    });
+  });
+
+  describe(`.logLevel (setter)`, function() {
+    it(`should allow valid log levels`, function() {
+      expect(() => {
+        logHelper.logLevel = logHelper.LOG_LEVELS.verbose;
+        logHelper.logLevel = logHelper.LOG_LEVELS.debug;
+        logHelper.logLevel = logHelper.LOG_LEVELS.warning;
+        logHelper.logLevel = logHelper.LOG_LEVELS.error;
+      }).to.not.throw();
+    });
+
+    // TODO: Catch WorkboxError with error code
+    it(`should not allow log level less than verbose`, function() {
+      expect(() => {
+        logHelper.logLevel = logHelper.LOG_LEVELS.verbose - 1;
+      }).to.throw();
+    });
+
+    // TODO: Catch WorkboxError with error code
+    it(`should not allow log level greater than error`, function() {
+      expect(() => {
+        logHelper.logLevel = logHelper.LOG_LEVELS.error + 1;
+      }).to.throw();
+    });
+
+    // TODO: Catch WorkboxError with error code
+    it(`should not allow non-number log levels`, function() {
+      expect(() => {
```

Comment:
Totally valid and awesome idea (Worth doing sooner rather than later).

I avoided that hear because I realised I complicated a number of the tests I had written but this seems like a nice and safe option.

See generate-variant-tests.js